src/sbvr-api/permissions.ts

```diff
@@ -950,18 +950,20 @@ const checkApiKey = Promise.method((req: PermissionReq, apiKey: string) => {
 	if (apiKey == null || req.apiKey != null) {
 		return;
 	}
-	return getApiKeyPermissions(apiKey)
-		.catch(err => {
-			console.warn('Error with API key:', err);
-			// Ignore errors getting the api key and just use an empty permissions object
-			return [];
-		})
-		.then(permissions => {
+	return Promise.join(
+		getApiKeyPermissions(apiKey),
+		getApiKeyActorId(apiKey),
+		(permissions: any, actor: number) => {
```

Comment:
If you remove the explicit typings then the types should be automatically inferred correctly (and will be more precise than `any`)

Reply:
Done